src/Tracking/TrackerSnapshot.php

```diff
@@ -77,18 +79,25 @@ protected function get_settings(): array {
 		/** @var TargetAudience $target_audience */
 		$target_audience = $this->container->get( TargetAudience::class );
 		$mc_settings     = $this->options->get( OptionsInterface::MERCHANT_CENTER );
+		/** @var AdsService $ads_service */
+		$ads_service = $this->container->get( AdsService::class );
+		/** @var SetupCampaignNote $campaign_setup */
+		$campaign_setup = $this->container->get( SetupCampaignNote::class );
 
 		return [
-			'version'          => $this->get_version(),
-			'db_version'       => $this->options->get( OptionsInterface::DB_VERSION ),
-			'tos_accepted'     => $this->get_boolean_value( OptionsInterface::WP_TOS_ACCEPTED ),
-			'google_connected' => $this->get_boolean_value( OptionsInterface::GOOGLE_CONNECTED ),
-			'mc_setup'         => $this->get_boolean_value( OptionsInterface::MC_SETUP_COMPLETED_AT ),
-			'ads_setup'        => $this->get_boolean_value( OptionsInterface::ADS_SETUP_COMPLETED_AT ),
-			'target_audience'  => $target_audience->get_target_countries(),
-			'shipping_rate'    => $mc_settings['shipping_rate'] ?? '',
-			'shipping_time'    => $mc_settings['shipping_time'] ?? '',
-			'tax_rate'         => $mc_settings['tax_rate'] ?? '',
+			'version'                         => $this->get_version(),
+			'db_version'                      => $this->options->get( OptionsInterface::DB_VERSION ),
+			'tos_accepted'                    => $this->get_boolean_value( OptionsInterface::WP_TOS_ACCEPTED ),
+			'google_connected'                => $this->get_boolean_value( OptionsInterface::GOOGLE_CONNECTED ),
+			'mc_setup'                        => $this->get_boolean_value( OptionsInterface::MC_SETUP_COMPLETED_AT ),
+			'ads_setup'                       => $this->get_boolean_value( OptionsInterface::ADS_SETUP_COMPLETED_AT ),
+			'target_audience'                 => $target_audience->get_target_countries(),
+			'shipping_rate'                   => $mc_settings['shipping_rate'] ?? '',
+			'shipping_time'                   => $mc_settings['shipping_time'] ?? '',
+			'tax_rate'                        => $mc_settings['tax_rate'] ?? '',
+			'has_account_issue'               => $campaign_setup->has_account_issues() ? 'yes' : 'no',
+			'has_at_least_one_synced_product' => $campaign_setup->has_at_least_one_synced_product() ? 'yes' : 'no',
+			'is_ads_setup_started'            => $ads_service->is_setup_started() ? 'yes' : 'no',
```

Comment:
We already have `ads_setup` so to me it would have been more logical to name it `ads_setup_started`. But not a big deal if it's requested to be named like that.

Reply:
I pretty much copied this key value from the issue ticket. Not sure if the name is set in stone. I will check with @jconroy

Reply:
`ads_setup_started` is fine 👍

Reply:
Sweet! updated!